contrib/setup-cvc5.sh

```diff
@@ -16,27 +17,22 @@ else
     NUM_CORES=1
 fi
 
-$DIR/setup-cadical.sh
+mkdir -p "$DEPS_DIR" && cd "$DEPS_DIR"
 
-if [ ! -d "$DEPS/cvc5" ]; then
-    cd $DEPS
-    git clone https://github.com/cvc5/cvc5.git
-    cd cvc5
-    git checkout -f ${CVC5_VERSION}
+if [ ! -d cvc5 ]; then
+    "$CONTRIB_DIR/setup-cadical.sh"
+    git clone https://github.com/cvc5/cvc5.git && cd cvc5
+    git checkout -f $CVC5_VERSION
     # ensure the cvc5 libraries are placed in deps/install/lib
     ./configure.sh --prefix=$DEPS/install --static --auto-download --dep-path="$DEPS/install" -DCMAKE_INSTALL_LIBDIR=lib
     cd build
-    make -j$NUM_CORES
-    make install
-    cd $DIR
+    cmake --build . --target install -j$NUM_CORES
```

Comment:
This is the main change.

Reply:
just wondering -- why shouldn't we install it according to the instructions of cvc5?

Reply:
I changed this so that we can use `ninja` instead of `make` transparently. I opened a PR in cvc5 too: https://github.com/cvc5/cvc5/pull/11879

Reply:
in that case, should we include `ninja` as a dependency?

Reply:
I added it to the `README`. Either Make or Ninja should work.